src/atomate2/utils/powerups.py

```diff
@@ -0,0 +1,68 @@
+"""Utilities for modifying workflow."""
```

Comment:
I would move these to the `src/atomate2/vasp/powerups.py` directory and set the default class filter to be `BaseVaspMaker`.

Reply:
Hi @utf , thinking a bit more about the location of this module, maybe is it not better to keep it more general? 

Like it could also be used for adding metadata/modify custodian handlers if for example other types of calculations for example Lobster runs, or cp2k, abinit etc in future?

Just wanted to know your 2nd opinion

Reply:
We could have a general implementation but then we should include a specific implementation for vasp/cp2k as well. I would also suggest that `atomate2/common/powerups.py` is a better place for the common implementation to live.

Then in `atomate2/vasp/powerups.py` you would do:

```python
from atomate2.common.powerups import add_metadata_to_flow as base_add_metadata_to_flow

def add_metadata_to_flow(...)
    base_add_metadata_to_flow(..., class_filter=BaseVaspMaker)
```

Obviously, you'll need to replace the `...` and add the docstrings.

Reply:
Done